<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogoEditarMedioFacebook</class>
 <widget class="QDialog" name="DialogoEditarMedioFacebook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>75</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editar cuenta de Facebook</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_campos" native="true">
     <layout class="QVBoxLayout" name="layout_campos">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_etiqueta" native="true">
        <layout class="QHBoxLayout" name="layout_etiqueta">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl_etiqueta">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Etiqueta</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineedit_etiqueta">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_nombre" native="true">
        <layout class="QHBoxLayout" name="layout_nombre">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl_nombre_pagina">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Nombre p√°gina</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineedit_nombre_pagina">
           <property name="font">
            <font>
             <family>Calibri</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QWidget" name="widget_opciones" native="true">
     <layout class="QHBoxLayout" name="layout_opciones">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_guardar">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Guardar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancelar">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
